src/windows/windows.js

```diff
@@ -1,46 +1,60 @@
 import F from '../functional';
-import { emptyDevice, emptyVolume, splitEOL } from '../utilities';
+import { emptyDevice, emptyVolume } from '../utilities';
+import csvSync from 'csv-parse/lib/sync';
 const { compose, reduce, filter } = F.R;
 
-export const COMMAND = 'wmic logicaldisk get ' +
-  'Caption,Description,DeviceID,FileSystem,FreeSpace,Name,Size,VolumeName';
+export const COMMAND =
+  'wmic logicaldisk get ' +
+  'Caption,Description,DeviceID,DriveType,FileSystem,FreeSpace,Name,Size,VolumeName ' +
+  '/format:csv';
 
-export const parseWindowsProps =
-(acc, [ caption, desc, id, filesystem, space, name, size, volumename ]) => {
-  acc.devices[name] = acc.devices[name] ? acc.devices[name] : emptyDevice();
-  acc.devices[name].id = id;
-  acc.devices[name].whole = true;
-  acc.devices[name].parent = id;
-  acc.devices[name].node = caption;
-  acc.devices[name].name = name;
-  acc.devices[name].size = parseInt(size);
-  acc.devices[name].description = desc;
+export const parseWindowsProps = (
+  acc,
+  {
+    Caption,
```

Comment:
Hmmm, personally style-wise I prefer to have parameter objects on a single line whenever possible. How many characters does it come up to? If you have a strong enough reason to put it that way, I don't mind moving forward with this. Let me know what you think

Reply:
Sorry about that; i use sublime text with jsprettier in all my nodejs projects, thats the why (by default it cuts lines greater than 80, and other defaults about splitting lines on parameter objects, etc). But i can leave them on the same line if you want, no problem!

Reply:
No worries, it happens :)